### ISSUE TITLE
"The library contains critical security issues"

### ISSUE THREAD

**Issue by 9mm** [nuxt-modules/i18n#2301]
### Environment

- Operating System: `Darwin`
- Node Version:     `v18.15.0`
- Nuxt Version:     `3.6.5`
- Nitro Version:    `2.5.2`
- Package Manager:  `pnpm@8.6.12`
- Builder:          `vite`
- User Config:      `app`, `device`, `devtools`, `image`, `modules`, `runtimeConfig`, `site`, `ssr`, `typescript`
- Runtime Modules:  `@nuxt/image@1.0.0-rc.1`, `@nuxtjs/device@3.1.0`, `@nuxtjs/i18n@8.0.0-rc.2`, `@nuxtjs/tailwindcss@6.8.0`, `@pinia/nuxt@0.4.11`, `nuxt-og-image@2.0.13`, `nuxt-schema-org@2.2.0`, `nuxt-simple-sitemap@3.1.5`
- Build Modules:    `-`

### Reproduction

There's no reproduction, as all I did was install using pnpm

### Describe the bug

```
pnpm add -D @nuxtjs/i18n@next
 WARN  deprecated vm2@3.9.19: The library contains critical security issues and should not be used for production! The maintenance of the project has been discontinued. Consider migrating your code to isolated-vm.
```

### Additional context

I searched issues to see if I could find anything, but I don't see any reference to this

**Comment by kazupon** [comment]
Thank you for your reporting!

Unfortunately, We could not your issue.
Could you give your minimal reproduction us with stackbliz ([nuxt starter](https://stackblitz.com/github/nuxt/starter/tree/v3-stackblitz)) or github repo please? 🙏

**Comment by huang-julien** [comment]
Hi :wave: i18n-module doesn't have vm2 as a dependency, can you check what makes you install `vm2` with `pnpm why vm2` please ? 🙏

**Comment by 9mm** [comment]
Sorry about that... I didnt realize pnpm dumped errors for every package to the console even when installing other packages. It was a problem in another library

```
┌─────────────────────┬────────────────────────────────────────────────────────┐
│ critical            │ vm2 Sandbox Escape vulnerability                       │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Package             │ vm2                                                    │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Vulnerable versions │ <=3.9.19                                               │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Patched versions    │ <0.0.0                                                 │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Paths               │ . > nuxt-og-image@2.0.13 > inline-css@4.0.2 >          │
│                     │ extract-css@3.0.1 > href-content@2.0.2 >               │
│                     │ remote-content@3.0.1 > superagent-proxy@3.0.0 >        │
│                     │ proxy-agent@5.0.0 > pac-proxy-agent@5.0.0 >            │
│                     │ pac-resolver@5.0.1 > degenerator@3.0.4 > vm2@3.9.19    │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ More info           │ https://github.com/advisories/GHSA-cchq-frgv-rjh5      │
└─────────────────────┴────────────────────────────────────────────────────────┘
```
